.travis.yml

```diff
@@ -3,11 +3,11 @@ sudo: false
 matrix:
     include:
     - python: "3.5"
-      env: DEPS="numpy=1.11.2 scipy=0.18.1 cython nose matplotlib"
+      env: DEPS="numpy=1.11.2 scipy=0.18.1 cython nose matplotlib scikit-learn=0.18.1"
     - python: "3.4"                                                                                         
-      env: DEPS="numpy=1.10.4 scipy=0.17 cython nose matplotlib"
+      env: DEPS="numpy=1.11.2 scipy=0.17 cython nose matplotlib scikit-learn=0.18.1"
```

Comment:
@rth Some good points there. Thanks.
Point 1 is very sensible. 
Point 2: This is what that Krige class is?
Point 3. the wrapper `Krige` class makes the pykrige classes scikit-learn compatible.
Point 4, you are spot on. That pipeline.py is just an example on how to use the Krige class. We can rename it something like you suggest.